.github/CODEOWNERS

```diff
@@ -394,8 +394,118 @@
+/x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/network @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/users @elastic/security-threat-hunting-explore
+
+## Security Solution sub teams - Detections and Response Alerts
+/x-pack/plugins/security_solution/server/lib/detection_engine/migrations @elastic/security-detections-response-alerts
+/x-pack/plugins/security_solution/server/lib/detection_engine/notifications @elastic/security-detections-response-alerts
+/x-pack/plugins/security_solution/server/lib/detection_engine/schemas @elastic/security-detections-response-alerts
+/x-pack/plugins/security_solution/server/lib/detection_engine/rule_types @elastic/security-detections-response-alerts
+/x-pack/plugins/security_solution/server/lib/detection_engine/signals @elastic/security-detections-response-alerts
+/x-pack/plugins/security_solution/server/lib/detection_engine/tags @elastic/security-detections-response-alerts
+
+## Security Solution sub teams - Detections and Response Rules
+
+/x-pack/plugins/security_solution/public/detections/components/rules @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/public/detections/components/severity @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/public/detections/components/status @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/public/rules @elastic/security-detections-response-rules
+
+/x-pack/plugins/security_solution/server/lib/detection_engine/rule_actions @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/server/lib/detection_engine/rule_execution_log @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/server/lib/detection_engine/rules @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/factories @elastic/security-detections-response-alerts
+
+
+## Security Solution sub teams - Security Platform
+/x-pack/plugins/lists @elastic/security-solution-platform
+
+/x-pack/plugins/security_solution/cypress/integration/data_sources @elastic/security-solution-platform
+/x-pack/plugins/security_solution/cypress/integration/exceptions @elastic/security-solution-platform
+/x-pack/plugins/security_solution/cypress/integration/value_lists @elastic/security-solution-platform
+
+/x-pack/plugins/security_solution/public/common/components/exceptions @elastic/security-solution-platform
+/x-pack/plugins/security_solution/public/exceptions @elastic/security-solution-platform
+
+/x-pack/plugins/security_solution/server/lib/sourcerer @elastic/security-solution-platform
+/x-pack/plugins/security_solution/server/lists_integration @elastic/security-solution-platform
+/x-pack/plugins/security_solution/server/routes @elastic/security-solution-platform
```

Comment:
This is probably better split between the rules and alerts area.

Reply:
Alerts area mostly takes on `routes/index/` and `routes/signals/`